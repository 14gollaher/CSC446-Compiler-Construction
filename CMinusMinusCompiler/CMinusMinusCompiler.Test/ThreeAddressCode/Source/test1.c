int a,b,c;
int main()
{
   a=10;
   b=5;
   c=a+b;
   return 0;
}
