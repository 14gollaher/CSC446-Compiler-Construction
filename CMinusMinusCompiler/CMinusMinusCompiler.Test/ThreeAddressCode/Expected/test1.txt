PROC main
	_BP-2 = 10
	_a = _BP-2
	_BP-4 = 5
	_b = _BP-4
	_BP-6 = _a + _b
	_c = _BP-6
	_BP-8 = 0
	_AX = _BP-8
ENDP main

START PROC main

